Source/Relativity.Test.Helpers/Relativity.Test.Helpers/Kepler/OAuth2Helper.cs

```diff
@@ -0,0 +1,164 @@
+﻿using kCura.Relativity.Client;
+using Newtonsoft.Json.Linq;
+using Relativity.Services.Security;
+using Relativity.Services.Security.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using DTOs = kCura.Relativity.Client.DTOs;
+
+namespace Relativity.Test.Helpers.Kepler
+{
+	public class OAuth2Helper : IOAuth2Helper
+	{
+		private readonly IOAuth2ClientManager _oAuth2ClientManager;
+		private readonly IRSAPIClient _rsapiClient;
+
+		public OAuth2Helper(IOAuth2ClientManager oAuth2ClientManager, IRSAPIClient rsapiClient)
+		{
+			_oAuth2ClientManager = oAuth2ClientManager;

```

Comment:
throw argument null exceptions when the arguments are null.

Reply:
Updated please review